src/views/comparison/ChartA.js

```diff
@@ -248,9 +248,10 @@ export class ChartA {
         .selectAll("patharea")
```

Comment:
I don't think `.selectAll("patharea")` is actually doing anything here (same for the other patharea or pathline lower in this file)